### DISCUSSION TITLE
"Open WezTerm here" question

### DISCUSSION THREAD

**Discussion by russarr** [wezterm/wezterm#4340] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

wezterm 20230712-072601-f4abf8fd

### Ask your question!

By default If i open wezterm using Explorer -> Right Mouse Button -> "Open WezTerm here" Wezterm opens with cwd in current directory.
I have configured the window settings:
```
wezterm.on("gui-startup", function()
  local tab, pane, window = wezterm.mux.spawn_window({
    position = {
      x = 1310,
      y = 100,
    },
    width = 150,
    height = 30,
  })
end)
```
And now "Open WezTerm here" opened always in my home directory. 
I tryed to set up `cwd="$PWD"`, but it doesn't help.
How to return set up currend directory behaviour in this case?

**Comment by wez** [comment]
the `gui-startup` event receives a `cmd` argument that will have the cwd set on it, which you can pass through to your explicit spawn call. 

Take a look at the examples on https://wezfurlong.org/wezterm/config/lua/gui-events/gui-startup.html to see how make use of it.
